nilearn/_utils/niimg.py

```diff
@@ -44,14 +44,14 @@ def load_img(img, dtype=None):
     img: image
```

Comment:
I guess this should be `niimg` and the type description should be Niimg-like object + URL